src/vi.h

```diff
@@ -193,6 +193,11 @@ static const vi_config_t vi_config_presets[2][3] = {
 /** @brief VI_BURST Register: PAL default horizontal sync width in pixels. */
 #define VI_HSYNC_WIDTH_PAL                  58
 
+/** @brief VI period for showing one NTSC and MPAL picture in ms. */
+#define VI_PERIOD_NTSC_MPAL                 (1000/60)
+/** @brief VI period for showing one PAL picture in ms. */
+#define VI_PERIOD_PAL                       (1000/50)
```

Comment:
I think these should be floating points, to avoid losing precision. Since they are going into vi.h which will be public one day, I'd rather export an accurate period for our users, rather than an approximated one.